samples/hosts/match_username_to_host.py

```diff
@@ -276,6 +277,10 @@ def upload_device_logins(aids: list):
         ]
 
     args = parse_command_line()
+      
+    if args.client_secret is None:
+        args.cliet_secret = getpass.getpass("ClientSecret: ") 
```

Comment:
Typo - should be: `args.client_secret`

Reply:
Line #59 will also need to be set to `False` or the argument parser will throw an error before we hit the if statement.